app/policies/value_definition_policy.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Policies for accessing Value Definitions
+class ValueDefinitionPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  # All users should see all value definitions currently
+  class Scope < ::ApplicationPolicy::Scope
+    def resolve
+      scope.all
+    end
+  end
```

Comment:
Not sure if these are the correct permissions 🤔 I figured since we only have the read-only endpoints this would be okay?

Reply:
I think it should be okay